test/typings/should_not_compile/index.ts

```diff
@@ -0,0 +1,26 @@
+import {createSelector} from '../../../src/reselect.d.ts';
+import * as common from '../common';
+
+
+const implicitlyTypedSelector = createSelector(
```

Comment:
I searched for this, but could find nothing. I just wrote a SO question about it. Maybe something comes from that: https://stackoverflow.com/questions/37927730/how-do-i-write-negative-tests-for-typescript-typings